scripts/test_unit.sh

```diff
@@ -6,6 +6,8 @@ if [ -z ${GO_ENV} ]; then
     export GO_ENV="test"
 fi
 
+export ATHENS_MINIO_ENDPOINT="127.0.0.1:9001"
```

Comment:
this should not be here, at least not with s3 pack.

Reply:
so should I create issue for that and make pull request with that? Because without this, storage tests for minio are ommited.